404.html.md

```diff
@@ -4,6 +4,8 @@ title: Page Not Found
 
 
 
+
```

Comment:
I have been using a very recent quarto daily. Let me know if there is a specific version that I should use.